src/tbbmalloc/backend.h

```diff
@@ -265,6 +265,7 @@ class Backend {
     IndexedBins freeLargeBlockBins,
                 freeSlabAlignedBins;
 
+    std::atomic<intptr_t> backendCleanCnt;
```

Comment:
it might be better to use unsigned integer to have a well-defined overflow

Reply:
It would be. But i kept this similar to other counters in which is for some reason also intptr_t